packages/amber/views/Reports.tsx

```diff
@@ -40,18 +42,20 @@ export const Reports: React.FC<ReportsProps> = ({ reports }) => {
         {filteredReports.map((r) => {
           const fileLabel = r?.fileLabel ?? toCamelCase(r.name)
           const url = r?.url ?? `${fileLabel}Report`
-          // console.log({ name: r.name, url })
-
-          return (
-            <ListItem key={url}>
-              <AuthenticatedDownloadButton
-                url={`/api/reports/${url}`}
-                filename={`${abbr}${configuration.year}-${fileLabel}-${timestamp}.xlsx`}
-              >
-                Download {r.name} Report
-              </AuthenticatedDownloadButton>
-            </ListItem>
-          )
+          if (r.perm === undefined || HasPermission(r.perm)) {
```

Comment:
I'm a bit surprised that this works, HasPermission is a component.

try:

```tsx
  const { hasPermissions } = useAuth()
```

at the beginning of the component, then:

```tsx
if (r.perm === undefined || hasPermission(r.perm)) {
```

Reply:
Shouldn't have. I'm bitterly disappointed in the TypeScript transpiler in letting me hornswoggle it.

Fixed it.